Agreements/Internal/TCDAG_12.txt

```diff
@@ -1,20 +1,8 @@
-COPX:EGLL:*:*:LEDBO:*:London TC DAGGA:London S12:23000:*:^TEDSA
-COPX:EGWU:*:*:LEDBO:*:London TC DAGGA:London S12:23000:*:^TEDSA
-COPX:EGLL:*:*:SOMVA:*:London TC DAGGA:London S12:23000:*:^PAAVO
-COPX:EGWU:*:*:SOMVA:*:London TC DAGGA:London S12:23000:*:^PAAVO
-COPX:EGLL:*:*:DAGGA:*:London TC DAGGA:London S12:23000:*:^GASBA
-COPX:EGWU:*:*:DAGGA:*:London TC DAGGA:London S12:23000:*:^GASBA
+;COPX for LL/WU/GW
+COPX:BRAIN:*:*:*:*:London TC DAGGA:London S12:23000:*:^BRAIN
```

Comment:
LL Traffic can also go via DIGSU

BPK-TOTRI-MATCH-DIGSU-TUZLA...

Reply:
Added!

(DIGSU as FIXBEFORE should cover both direct UMBAG and P5 as you've commented)

Reply:
FIXBEFORE MATCH will work too and is probably a more elegant solution. This will match (pun not intended!) some routes out of smaller aerodromes like:
- TB (Wycombe) - DCT BPK Q295 BRAIN P44 CLN L620 REDFA
- TC (Cranfield) DCT BPK Q295 BRAIN M197 REDFA 
- TR (Elstree) DCT BPK Q295 BRAIN M197 REDFA 
- UB (Benson) DCT WCO DCT BPK Q295 BRAIN P44 CLN M84 KONAN _and_ DCT WCO DCT BPK Q295 BRAIN M197 REDFA

These routes are all very similar to the GW/LL/WU ones so I suspect that it would be the intended behaviour that the agreement applies to these?

Everything else either leaves Q295 at BPK (routing N16 BKY or UM185 to DIGSU) or joins Q295 at BRAIN from M197 (from CPT or OCK) -- leaving the only traffic via MATCH the outbounds we want to... match!

Reply:
@cpawley is it ok for this agreement to display for outbounds from these fields, or do they need to be coordinated / does it need to be put in documentation first?

Reply:
They will be displayed either way as they also match 
`COPX:BRAIN:*:*:*:*:London TC DAGGA:London S12:23000:*:^BRAIN`

Reply:
Fairs, let's do it then! It makes sense on VATSIM... we're generally not good at coordinating things, and they're so infrequent it needn't be on documentation.

Reply:
I imagine the coordination challenges are more relevant in TC North airspace where they join CAS. By the time they're with TC DAGGA I suspect they'd be indistinguishable from LL/GW deps.